ocw/lib/EC2.py

```diff
@@ -72,6 +76,24 @@ def eks_client(self, region):
                                                      region_name=region)
         return self.__eks_client[region]
 
+    def kubectl_client(self, region, cluster_name):
+        region_cluster = f"{region}/{cluster_name}"
+
+        if region_cluster not in self.__kubectl_client:
+            kubeconfig = f"~/.kube/config_{region}_{cluster_name}"
```

Comment:
```suggestion
            kubeconfig = f"~/.kube/eks_config_{region}_{cluster_name}"
```

I think in next iteration you will store AKS/GKE configs in same dir

Reply:
good point!

Reply:
done